### PR TITLE
[PLATFORM-182] Added the metadata root address

### PR DESCRIPTION
**Pull request by Kanan-Bitrise**
### Pull Request Checklist

Check/accept these:

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [x] I've updated the version in the Dockerfile
- [x] I've updated the CHANGELOG.md
- [ ] I've provided a link or explanation below which describes the changes in the tool itself
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added

`Copy or link the tool's changelog here`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,7 @@ FROM quay.io/bitriseio/android:alpha
 
 ENV ANDROID_NDK_HOME /opt/android-ndk
 ENV ANDROID_NDK_VERSION r21c
+ENV GCE_METADATA_ROOT 127.0.0.1
```
Comment (peterheja-bitrise) [inline Dockerfile:5]: Why do we set this? What do we gain from this?
I see no real reason to override this. Are we trying to break these metadata calls or what?

**Comment by bitrisekristof** [comment]
@bitrise-tom @peterheja-bitrise https://bitrise.atlassian.net/browse/PLATFORM-182 (check the comments too)

TLDR: The google sends metadata to the VM's anyway, and we have to do this as a workaround, because the gcloud thinks he is en our project, and the users can't do anything with gcloud if they are not changing it explicitly, even they use the correct service json.

**Comment by bitrise-tom** [comment]
I am concerned that this is a sticking plaster fix and has the potential to break other things in the container

**Comment by bitrisekristof** [comment]
This why I would like to run tests for it.

**Comment by Kanan-Bitrise** [comment]
Indeed
